### PR TITLE
Update ZIO to 2.0.0-RC5

### PR DESCRIPTION
**Pull request by alphaho**

### CODE REVIEW

File: src/main/scala/zio/interop/guava/package.scala [hunk 1]
```diff
@@ -106,12 +98,12 @@ package object guava {
               UIO.succeedNow(None)
           }
 
-        final def getRef[A](ref: FiberRef.Runtime[A])(implicit trace: ZTraceElement): UIO[A] = ref.get
+        final def getRef[A](ref: FiberRef[A])(implicit trace: ZTraceElement): UIO[A] = ref.get
```
Comment (alphaho) [inline src/main/scala/zio/interop/guava/package.scala:101]: I'm not exactly sure whether this change is correct or not, as I'm a bit lost in FiberRef's changes between ZIO 2.0.0-RC2 and 2.0.0-RC5

**Comment by sideeffffect** [comment]
Thanks for helping out with this!
The CI is red, please have a look at that :pray:

**Comment by alphaho** [comment]
It turns out the `getRef` method is removed from the interface, making it get `Unused method` error. Have now removed the method and hopefully it would fix the build.

**Review by sideeffffect — APPROVED** [review]
Thanks! :heart:

**Comment by juliano** [comment]
Hey guys! Do you intend to merge this one and make a new release soon? I am using this dependency heavily and would need it to update my stack. Thanks!

**Comment by sideeffffect** [comment]
Sorry, I don't know why I haven't merged and released it right away :sweat_smile: 
There you go guys https://github.com/zio/interop-guava/releases/tag/v31.0.1.1 I'm happy somebody is using this! :grin:

**Pull request merged**